CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 0.4.1-dev (changes since 0.4.1 go here)
 
 ### Features
+* Added `skbio.tree.TreeNode.bifurcate` for converting multifurcating trees into bifurcating trees. [#896](https://github.com/biocore/scikit-bio/issues/896))
```

Comment:
:L

Fixed

On Wed, Feb 17, 2016 at 10:23 AM, Jai Ram Rideout <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/biocore/scikit-bio/pull/904#discussion_r53207287>:
>
> > @@ -3,6 +3,7 @@
> >  ## Version 0.4.1-dev (changes since 0.4.1 go here)
> >
> >  ### Features
> > +* Added `skbio.tree.TreeNode.bifurcate` for converting multifurcating trees into bifurcating trees. [#896](https://github.com/biocore/scikit-bio/issues/896))
>
> The issue number doesn't render correctly, can you add an opening paren?
>
> [image: screen shot 2016-02-17 at 11 21 32 am]
> <https://cloud.githubusercontent.com/assets/1847232/13119904/a31325e0-d568-11e5-8f6e-b1822727b921.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/904/files#r53207287>.
>